Dockerfile

```diff
@@ -0,0 +1,14 @@
+FROM node:14-alpine
```

Comment:
I think we can safely go down to 12 since its the latest LTS release unless we absolutely need a feature from 14.

Reply:
Happy to downgrade to 12, but I noticed in Excalidraw repo we were using 14